### ISSUE TITLE
Add support for 'context' keyword to translate()

### ISSUE THREAD

**Issue by piotr-dobrogost** [Pylons/translationstring#27]
translate() should support 'context' keyword analogically to how it supports 'domain' keyword.
Till then one can workaround this by wrapping string to translate in extra `TranslationString` like so:
`translate(TranslationString(tstring, context='some_ctx'))`

**Comment by ztane** [comment]
+1, and same applies of course to pluralization (Pluralizer) everywhere as well

**Comment by wichert** [comment]
`translate()` already [has a context method](https://github.com/Pylons/translationstring/blob/master/translationstring/__init__.py#L185) ?

**Comment by ztane** [comment]
Wrong link, should be [this](https://github.com/Pylons/translationstring/blob/master/translationstring/__init__.py#L306) but you're correct.

**Comment by ztane** [comment]
But docs are not up-to-date: http://docs.pylonsproject.org/projects/translationstring/en/latest/translation.html

**Comment by ztane** [comment]
And here http://docs.pylonsproject.org/projects/translationstring/en/latest/api.html#translationstring.Translator

**Comment by mmerickel** [comment]
Yes it looks like a few of the docstrings are out of date. A PR to fix these would be great.
